beta-3-setup.md

```diff
@@ -2922,3 +2923,248 @@ HTTPS_PROXY=https://USER:PASSWORD@IPADDR:PORT
+
+### Add the desired lifecycle hook
+
+The hook should run the command with the appropriate environment in
+the correct directory, like:
+
+    /usr/bin/scl enable ruby200 ror40 'cd /opt/openshift/src ; bundle exec rake db:migrate'
+
+...but expressed as a string array in `json`:
+
+    "command": [
+        "/usr/bin/scl",
+        "enable",
+        "ruby200",
+        "ror40",
+        "cd /opt/openshift/src ; bundle exec rake db:migrate"
+    ]
+
+#### create/modify the `recreateParams` `post` hook definition for the template strategy
+
+Don't forget to copy & paste the `env` variable definitions from
+the `env.json` file you created earlier. You should end up with a
+section that looks like:
+
+    ...
+        "template": {
+            "strategy": {
+                "type": "Recreate",
+                "recreateParams": {
+    
+    ...
+                    "post": {
+                        "failurePolicy": "Ignore",
+                        "execNewPod": {
+                            "command": [
+                                "/usr/bin/scl",
+                                "enable",
+                                "ruby200",
+                                "ror40",
+                                "cd /opt/openshift/src ; bundle exec rake db:migrate"
+                            ],
+                            "env": [
+                                {
+                                    "name": "MYSQL_USER",
+                                    "value": "userJKL"
+                                },
+                                {
+                                    "name": "MYSQL_PASSWORD",
+                                    "value": "5678efgh"
+                                },
+                                {
+                                    "name": "MYSQL_DATABASE",
+                                    "value": "root"
+                                }
+                            ],
+                            "containerName": "ruby-helloworld"
+                        }
+                    }
+    ...
+
+## Kick off the new build:
+
+    osc start-build ruby-sample-build
+
+Monitor the build logs, and note the docker image tag, which should
+be output towards the end of the build on a line similar to this:
+
+    2015-04-23T19:59:34.304730547Z I0423 15:59:34.303984       1 sti.go:236] Tagged e696bbc88d892473a593b4e074483888e696bbc88d892473a593b4e074483888 as 172.30.17.99:5000/myapp2/origin-ruby-sample
+
+Check for that image tag in the output of `docker images`:
+
+    # docker images | grep e696bbc88d89
+    172.30.17.43:5000/myapp2/origin-ruby-sample                                        e696bbc88d892473a593b4e074483888e696bbc88d892473a593b4e074483888   e696bbc88d89        37 seconds ago      449.2 MB
+
+**Note:** It may take a few seconds/minutes for the image to appear,
+ and it may disappear again for a moment then reappear again. I
+ don't know why, but once it reappears, you're good to go.
```

Comment:
This is definitely weird.  I would leave it out and maybe track it as an issue instead.